trunk/Admin/AdminQuickbarAdmin.php

```diff
@@ -230,13 +232,68 @@ public function getLoopPosts( $postType, $posts ) {
                 'permalink' => $permalink,
                 'hasSwift' => $this->hasSwift,
                 'activeClass' => $activeClass,
+                'languageFlag' => $languageFlag,
+            ] );
+            $output .= $template->getRendered();
+        }
+
+        return $output;
+    }
+
+    /**
+     * @param $post
+     * @return string
+     */
+    public function getLanguageFlag( $post ) {
+        if ( !defined( 'ICL_LANGUAGE_CODE' ) ) {
+            return '';
+        }
+
+        global $sitepress;
+        $wpmlLanguageInfo = apply_filters( 'wpml_post_language_details', null, $post->ID );
+        $languageCode = $wpmlLanguageInfo['language_code'];
+        $flagUrl = $sitepress->get_flag_url( $languageCode );
+
+        $template = new Template( self::PartialDir . '/language-flag.php', [
```

Comment:
Are all of your constants in PascalCase ?

Reply:
PartialDir is the only constant which is not auto-generated, so yes, all constants are PascalCase.

Reply:
Ugh, how terrible

Reply:
Did you prefere `PARTIAL_DIR` ?